Monika After Story/game/script-topics.rpy

```diff
@@ -3221,6 +3224,55 @@ label monika_lovecounter_aff:
 
 default persistent._mas_last_monika_ily = None
 
+label monika_ilym_fight_start:
+    $ ilym_times_till_win = renpy.random.randint(6,10)
+    $ ilym_count = 0
+    $ ilym_quip = "I love you more, " + player + "!"
+    m 3hubfb "[ilym_quip]"
+    jump monika_ilym_fight
+
+label monika_ilym_fight:
+    $ _history_list.pop()
+    menu:
+        m "[ilym_quip]{fast}"
+        "No, I love you more!":
+            if ilym_count < ilym_times_till_win:
+                if renpy.random.randint(1,3) == 1:
+                    python:
+                        ilym_quips = [
+                            "No, I love you waaaaaaaaay more!",
+                            "No, I definitely love you more!",
+                            "Nuh-uh, I love you more!",
+                            "No no no, I love you more!"
+                        ]
+                        ilym_quip = renpy.random.choice(ilym_quips)
+                else:
+                    $ ilym_quip = "No, I love you more!"
+
+                $ ilym_exprs = ('1tubfb', '3tubfb', '1tubfu', '3tubfu', '1hubfb', '3hubfb', '1tkbfu')
+                $ renpy.show('monika ' + renpy.random.choice(ilym_exprs), at_list=[t11], zorder=MAS_MONIKA_Z)
+                m "[ilym_quip]"
+                $ ilym_count += 1
+                jump monika_ilym_fight
+
+            else:
+                m 5hubfb "Okay, okay, you win ahaha~"
```

Comment:
For pose 5 need to use dissolve like this
```suggestion
                show monika 5hubfb at t11 zorder MAS_MONIKA_Z with dissolve
                m 5hubfb "Okay, okay, you win ahaha~"
```

Reply:
I was using dissolve before but I didn't like the pause it makes before she says you win, it kind of spoils the surprise of winning since you know that's the only time the dissolve will happen. But if it's a requirement maybe instead using pose 2hkb with no dissolve would be better?